packages/react-instantsearch/src/components/LinkList.js

```diff
@@ -53,6 +53,7 @@ export default class LinkList extends Component {
               item.disabled && 'itemDisabled',
               item.modifier
             )}
+            disabled={item.disabled}
```

Comment:
Just discovered this, will use it in the previous PR about disabled widget states :)